### PR TITLE
Update crossplane docs link, add note to install.md to README

### PR DESCRIPTION
**Pull request by leonardpahlke**
Signed-off-by: leonardpahlke <leonard.pahlke@googlemail.com>

### Description of your changes

I have: updated the README to link to the crossplane docs page, which currently routes to 404 and added a sentence mentioning the `Install.md` file. ~Update PR template; `make reviewable test` - `make test` does not exist as command~.

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable` to ensure this PR is ready for review.

### How has this code been tested

not applicable

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Fixes #
 I have:
 
 - [ ] Read and followed Crossplane's [contribution process].
-- [ ] Run `make reviewable test` to ensure this PR is ready for review.
```
Comment (ADustyOldMuffin) [inline .github/PULL_REQUEST_TEMPLATE.md:25]: Please remove this change as we require that tests pass for changes to be approved.

**Review by ADustyOldMuffin — COMMENTED** [review]
Left a comment for you to address. Thanks for the quick update!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (leonardpahlke) [reply .github/PULL_REQUEST_TEMPLATE.md:25]: Yes, sure! But, one question- if I run `make test` I get the response: `make: *** No rule to make target `test'.  Stop.` and I cannot find a reference in the Makefile defining the cmd.

Comment (ADustyOldMuffin) [reply .github/PULL_REQUEST_TEMPLATE.md:25]: Make sure you initialize and pull the submodules! I believe it's in the submod for Crossplane's common build utilities.

**Review by ADustyOldMuffin — APPROVED** [review]

**Pull request merged**